perfkitbenchmarker/aws/aws_network.py

```diff
@@ -151,21 +151,27 @@ def _Delete(self):
 class AwsSubnet(resource.BaseResource):
   """An object representing an Aws subnet."""
 
-  def __init__(self, zone, vpc_id):
+  def __init__(self, zone, vpc_id, cider_block=None):
```

Comment:
`cidr_block`?

Can you make the default value `10.0.0.0/24` rather than `None` and setting it in `_Create`?